docs/source/intro-clx-streamz.ipynb

```diff
@@ -39,140 +39,167 @@
-    "--poll_interval 1s \\\n",
-    "--max_batch_size 1000 \\\n",
-    "--data /path/to/dataset\n",
+    "docker run -it \\\n",
+    "    -p 8787:8787 \\\n",
+    "    -p 8888:8888 \\\n",
+    "    -v <your_volume_binding_host_directory_path>:<your_volume_binding_container_directory_path> \\\n",
+    "    --gpus '\"device=0,1,3\"' \\\n",
+    "    --name clx_streamz \\\n",
+    "    -d clx-streamz:latest\n",
     "```\n",
     "\n",
     "**Legacy - Docker CE v18 and nvidia-docker2**\n",
     "```\n",
     "docker run -it \\\n",
-    "--runtime=nvidia \\\n",
-    "-p 8787:8787 \\\n",
-    "--name cybert-streamz\n",
-    "-d cybert-streamz:latest \\\n",
-    "--broker localhost:9092 \\\n",
-    "--group_id streamz \\\n",
-    "--input_topic input \\\n",
-    "--output_topic output \\\n",
-    "--model_file /path/to/model.pth \\\n",
-    "--label_file /path/to/label.json \\\n",
-    "--poll_interval 1s \\\n",
-    "--max_batch_size 1000 \\\n",
-    "--data /path/to/dataset\n",
+    "    -p 8787:8787 \\\n",
+    "    -p 8888:8888 \\\n",
+    "    -v <your_volume_binding_host_directory_path>:<your_volume_binding_container_directory_path> \\\n",
+    "    --runtime=nvidia \\\n",
+    "    --name clx_streamz \\\n",
+    "    -d cybert-streamz:latest\n",
     "```\n",
     "\n",
-    "**Parameters:**\n",
-    "- `broker`* - Host and port where kafka broker is running\n",
-    "- `group_id`* - Kafka [group id](https://docs.confluent.io/current/installation/configuration/consumer-configs.html#group.id) that uniquely identifies the streamz data consumer.\n",
-    "- `input_topic` - The name for the input topic to send the input dataset. Any name can be indicated here.\n",
-    "- `output_topic` - The name for the output topic to send the output data. Any name can be indicated here.\n",
-    "- `model_file` - The path to your model file\n",
-    "- `label_file` - The path to your label file\n",
-    "- `poll_interval`* - Interval (in seconds) to poll the Kafka input topic for data\n",
-    "- `max_batch_size`* - Max batch size of data (max number of logs) to ingest into streamz with each `poll_interval`\n",
-    "- `data` - The input dataset to use for this streamz example. This is a filepath to text file containing lines of text to be processed for inference (i.e. log file)\n",
-    "\n",
-    "``*`` = More information on these parameters can be found in the streamz [documentation](https://streamz.readthedocs.io/en/latest/api.html#streamz.from_kafka_batched).\n",
+    "The Dockerfile contains an ENTRYPOINT which calls [entrypoint.sh](https://github.com/rapidsai/clx/blob/branch-0.17/examples/streamz/scripts/entrypoint.sh) to:\n",
+    "1. Configure and install Kafka\n",
+    "2. Run Kafka broker on `localhost:9092` and Zookeeper on `localhost:2181`\n",
+    "3. Start Dask Scheduler on `localhost:8786`\n",
+    "4. Start Dask CUDA Worker (one worker per GPU)\n",
+    "\n",
+    "**Steps to run workflow**\n",
+    "\n",
+    "1. Access docker container\n",
```

Comment:
If it's possible, I'd avoid asking the user to log into the container. Might be better to pass any commands from host to the container instead.

Reply:
sure, I will make changes accordingly.

Reply:
done